<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="13" passed="7" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-17T04:00:38Z" name="Surefire suite" finished-at="2024-10-17T04:01:02Z" duration-ms="23958">
    <groups>
    </groups>
    <test started-at="2024-10-17T04:00:38Z" name="Surefire test" finished-at="2024-10-17T04:01:02Z" duration-ms="23958">
      <class name="userRequestTest">
        <test-method signature="createUser()[pri:1, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:38Z" name="createUser" finished-at="2024-10-17T11:00:44Z" duration-ms="5591" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method signature="getUser()[pri:2, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:46Z" name="getUser" finished-at="2024-10-17T11:00:48Z" duration-ms="1924" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getUser -->
        <test-method signature="updateUser()[pri:3, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:50Z" name="updateUser" finished-at="2024-10-17T11:00:51Z" duration-ms="1600" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateUser -->
        <test-method signature="loginUser()[pri:4, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:53Z" name="loginUser" finished-at="2024-10-17T11:00:55Z" duration-ms="1974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginUser -->
        <test-method signature="deleteUser()[pri:5, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:57Z" name="deleteUser" finished-at="2024-10-17T11:00:58Z" duration-ms="1724" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUser -->
        <test-method signature="logoutUser()[pri:6, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:01:00Z" name="logoutUser" finished-at="2024-10-17T11:01:02Z" duration-ms="1733" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutUser -->
      </class> <!-- userRequestTest -->
      <class name="petRequestTest">
        <test-method is-config="true" signature="SetUpSuite()[pri:0, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:37Z" name="SetUpSuite" description="Configure URI" finished-at="2024-10-17T11:00:38Z" duration-ms="1522" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUpSuite -->
        <test-method is-config="true" signature="SetUpSuite()[pri:0, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:00:38Z" name="SetUpSuite" description="Configure URI" finished-at="2024-10-17T11:00:38Z" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUpSuite -->
        <test-method signature="addPetTest()[pri:1, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:44Z" name="addPetTest" finished-at="2024-10-17T11:00:46Z" duration-ms="1844" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addPetTest -->
        <test-method signature="findPetByIdTest()[pri:2, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:48Z" name="findPetByIdTest" finished-at="2024-10-17T11:00:49Z" duration-ms="1642" status="FAIL">
          <exception class="groovy.lang.GroovyRuntimeException">
            <message>
              <![CDATA[could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"]]>
            </message>
            <full-stacktrace>
              <![CDATA[groovy.lang.GroovyRuntimeException: could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"at org.codehaus.groovy.reflection.CachedConstructor.createException(CachedConstructor.java:103)at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:89)at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:238)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:656)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:169)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:125)at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:133)at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)at petRequestTest.findPetByIdTest(petRequestTest.java:49)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)at java.base/java.lang.Thread.run(Thread.java:1589)]]>
            </full-stacktrace>
          </exception> <!-- groovy.lang.GroovyRuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findPetByIdTest -->
        <test-method signature="updatePetTest()[pri:3, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:51Z" name="updatePetTest" finished-at="2024-10-17T11:00:53Z" duration-ms="1905" status="FAIL">
          <exception class="groovy.lang.GroovyRuntimeException">
            <message>
              <![CDATA[could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <405>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"]]>
            </message>
            <full-stacktrace>
              <![CDATA[groovy.lang.GroovyRuntimeException: could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <405>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"at org.codehaus.groovy.reflection.CachedConstructor.createException(CachedConstructor.java:103)at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:89)at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:656)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:169)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:125)at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:133)at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)at petRequestTest.updatePetTest(petRequestTest.java:80)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)at java.base/java.lang.Thread.run(Thread.java:1589)]]>
            </full-stacktrace>
          </exception> <!-- groovy.lang.GroovyRuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updatePetTest -->
        <test-method signature="findPetByStatusTest()[pri:4, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:55Z" name="findPetByStatusTest" finished-at="2024-10-17T11:00:57Z" duration-ms="1477" status="FAIL">
          <exception class="groovy.lang.GroovyRuntimeException">
            <message>
              <![CDATA[could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"]]>
            </message>
            <full-stacktrace>
              <![CDATA[groovy.lang.GroovyRuntimeException: could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"at org.codehaus.groovy.reflection.CachedConstructor.createException(CachedConstructor.java:103)at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:89)at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:656)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:169)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:125)at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:133)at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)at petRequestTest.findPetByStatusTest(petRequestTest.java:92)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)at java.base/java.lang.Thread.run(Thread.java:1589)]]>
            </full-stacktrace>
          </exception> <!-- groovy.lang.GroovyRuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findPetByStatusTest -->
        <test-method signature="deletePetTest()[pri:5, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:00:58Z" name="deletePetTest" finished-at="2024-10-17T11:01:00Z" duration-ms="1800" status="FAIL">
          <exception class="groovy.lang.GroovyRuntimeException">
            <message>
              <![CDATA[could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"]]>
            </message>
            <full-stacktrace>
              <![CDATA[groovy.lang.GroovyRuntimeException: could not access constructor: private java.lang.AssertionError(java.lang.String) with arguments: [1 expectation failed.Expected status code <200> but was <404>.] reason: java.lang.IllegalAccessException: class org.codehaus.groovy.reflection.CachedConstructor cannot access a member of class java.lang.AssertionError (in module java.base) with modifiers "private"at org.codehaus.groovy.reflection.CachedConstructor.createException(CachedConstructor.java:103)at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:89)at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:77)at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:250)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:494)at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:656)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:210)at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:59)at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:169)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:125)at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:133)at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:119)at petRequestTest.deletePetTest(petRequestTest.java:103)at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)at java.base/java.lang.reflect.Method.invoke(Method.java:578)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)at java.base/java.lang.Thread.run(Thread.java:1589)]]>
            </full-stacktrace>
          </exception> <!-- groovy.lang.GroovyRuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deletePetTest -->
        <test-method is-config="true" signature="TearDownSuite()[pri:0, instance:petRequestTest@6d3a388c]" started-at="2024-10-17T11:01:02Z" name="TearDownSuite" finished-at="2024-10-17T11:01:02Z" duration-ms="23" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownSuite -->
        <test-method is-config="true" signature="TearDownSuite()[pri:0, instance:userRequestTest@223aa2f7]" started-at="2024-10-17T11:01:02Z" name="TearDownSuite" finished-at="2024-10-17T11:01:02Z" duration-ms="28" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDownSuite -->
      </class> <!-- petRequestTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
